includes/TypesForMembers/MemberRotationalRestraint.js

```diff
@@ -0,0 +1,361 @@
+ * @param {Number} sheeting_distance_of_ribs    Sheeting distance of ribs, can be undefined (default value taken from selected section)
+ * @param {Number} width_of_section_flange      Width of sheeting flange, can be undefined (default value taken from selected section)
+ * @param {Number} spring_stiffness             Spring stiffness, can be undefined (different depending on loading as default)
+ * @param {Number} beam_spacing                 Beam spacing, can be undefined (3 m as default)
+ */
+MemberRotationalRestraint.prototype.SetContinuousParameters = function (modulus_of_elasticity,
+    sheeting_thickness,
+    sheeting_moment_of_inertia,
+    sheeting_distance_of_ribs,
+    width_of_section_flange,
+    spring_stiffness,
+    beam_spacing) {
+    ASSERT(this.member_rotational_restraint.type === member_rotational_restraints.TYPE_CONTINUOUS, "Only for Continuous Member Rotational Restraint");
+    if (typeof modulus_of_elasticity !== "undefined") {
+        this.member_rotational_restraint.modulus_of_elasticity = modulus_of_elasticity;
+    }
+    if (typeof sheeting_thickness !== "undefined") {
+        this.member_rotational_restraint.sheeting_thickness = sheeting_thickness;
+    }
+    if (typeof sheeting_moment_of_inertia !== "undefined") {
+        this.member_rotational_restraint.sheeting_moment_of_inertia = sheeting_moment_of_inertia;
+    }
+    if (typeof sheeting_distance_of_ribs !== "undefined") {
+        this.member_rotational_restraint.sheeting_distance_of_ribs = sheeting_distance_of_ribs;
+    }
+    if (typeof width_of_section_flange !== "undefined") {
+        this.member_rotational_restraint.width_of_section_flange = width_of_section_flange;
+    }
+    if (typeof spring_stiffness !== "undefined") {
+        this.member_rotational_restraint.different_spring_stiffness = false;
+        this.member_rotational_restraint.spring_stiffness = spring_stiffness;
+    }
+    else {
+        this.member_rotational_restraint.different_spring_stiffness = true;
+    }
+    if (typeof beam_spacing !== "undefined") {
+        this.member_rotational_restraint.beam_spacing = beam_spacing;
+    }
+};
+
+/**
+ * Sets parameters for Discrete Member Rotational Restraint
+ * @param {Number}  modulus_of_elasticity       Modulus of elasticity, can be undefined (default value taken from selected sheeting material)
+ * @param {Number}  section_moment_of_inertia   Section moment of inertia, can be undefined (default value taken from selected section)
+ * @param {Number}  purlin_spacing              Purling spacing
+ * @param {Number}  beam_spacing                Beam spacing, can be undefined (3 m as default)
+ * @param {Number}  rotational_stiffness_value  Rotational stiffness, can be undefined (only if rotational stiffness is INFINITELY)
+ */
+MemberRotationalRestraint.prototype.SetDiscreteParameters = function (modulus_of_elasticity,
+    section_moment_of_inertia,
```

Comment:
depending on the sheeting - delete it

Reply:
But the values are enabled to edit/change.